core/src/main/scala/org/apache/gearpump/util/ActorUtil.scala

```diff
@@ -97,4 +99,23 @@ object ActorUtil {
       master.tell(StartExecutorSystems(resources, executorJvmConfig), sender)
     }
   }
- }
+
+  implicit val timeout = Constants.FUTURE_TIMEOUT
+  def sendMessageToAppMaster[T](master: ActorRef, appId: Int, msg: Any)(implicit ex: ExecutionContext): Future[T] = {
```

Comment:
better separate the logic to fetch ActorRef of `AppMaster`. Also, why not reuse `sendMessageToMaster` here ?